modules/gdscript/gdscript_function.h

```diff
@@ -234,6 +234,7 @@ class GDScriptFunctionState : public Reference {
 	GDScriptFunction *function;
 	GDScriptFunction::CallState state;
 	Variant _signal_callback(const Variant **p_args, int p_argcount, Variant::CallError &r_error);
+	GDScriptFunctionState* previousState;
```

Comment:
Godot's C++ code uses `snake_case` for all identifiers, should be `previous_state`.